### PR TITLE
Add basic 'howto' on adding CUBA keywords

### PR DESCRIPTION
**Pull request by nathanfranklin**
This PR:
- adds basic instructions on adding CUBA keywords (#209)

### CODE REVIEW

**Comment by nathanfranklin** [comment]
This is ready for review.

@ahashibon , does this text match the goal of #247

File: doc/source/extending_simphony.rst [hunk 1]
```diff
@@ -51,3 +54,32 @@ Will allow the user to import the new engine from inside the ``simphony`` module
...
+-------------
+
+Common Unified Base Attributes (CUBA) are a list of common keywords transcending
+across different scales, methods and modelling-engines. As SimPhoNy is extended,
+there is a need for CUBA to also be extended. Before adding a CUBA, developers
+should first consult the list of existing :ref:`cuba-keywords`,
+to see if the keyword already exists. Contact the developers at
+https://github.com/simphony/simphony-common if there are any questions.
+
+Once you have decided upon the what CUBA needs to be added (or modified), edit
```
Comment (nathanfranklin) [inline doc/source/extending_simphony.rst]: remove `the`

**Comment by nathanfranklin** [comment]
@mehdisadeghi , @ahashibon @TobiasRasp ,  if you have available time, can someone take a look at this. thanks.

File: doc/source/extending_simphony.rst [hunk 2]
```diff
@@ -51,3 +54,32 @@ Will allow the user to import the new engine from inside the ``simphony`` module
 
 
 .. _entry points : http://pythonhosted.org/setuptools/pkg_resources.html#entry-points
+
+
+CUBA keywords
+-------------
+
+Common Unified Base Attributes (CUBA) are a list of common keywords transcending
```
Comment (ahashibon) [inline doc/source/extending_simphony.rst]: Common Unified Basic Attributes

**Comment by ahashibon** [comment]
looks good! only a small comment, basic instead of base...

**Comment by nathanfranklin** [comment]
@ahashibon , addressed the spelling problem.

**Pull request merged**